cmake/configs/default.cmake

```diff
@@ -62,7 +62,7 @@ hunter_config(Libcxxabi VERSION ${HUNTER_Clang_VERSION})
 hunter_config(Libssh2 VERSION 1.7.0)
 hunter_config(Lua VERSION 5.3.2)
 hunter_config(MySQL-client VERSION 6.1.6)
-hunter_config(OpenCV VERSION 3.1.0-p1)
+hunter_config(OpenCV VERSION 3.1.0-p2 CMAKE_ARGS WITH_QT=YES)
```

Comment:
Yes, for non iOS, Android and WinRT it would trigger Qt5 base component by default as per OpenCV's CMakeLists.txt:
`OCV_OPTION(WITH_QT             "Build with Qt Backend support"               OFF  IF (NOT ANDROID AND NOT IOS AND NOT WINRT) )`

My reasoning behind setting it by default is that the Qt backend is enabled by default when downloading OpenCV from the site and when downloading highgui module on the average linux distribution. So it would create a more seamless transition to hunter.

But I do understand the burden of having to compile Qt, as I've been testing this package, so I'm also happy with an entry in the wiki about it.

Your call.